eventloop_unix.go

```diff
@@ -256,6 +256,17 @@ func (el *eventloop) read(c *conn) error {
 		return nil
 	}
 
+	if el.engine.opts.SuspendReaderWhenBufferFull && c.inboundBuffer.IsFull() {
```

Comment:
`inboundBuffer.IsFull` is true only when the current data size is equal to the capacity of the buffer, but `inboundBuffer` will grow when new data is written. So this method doesn't fit here.

Reply:
Yeah, I find out CPU would goes high when many slow connections come.